activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -2651,21 +2651,6 @@ void should_migrateTaskProcessVariables() throws Exception {
         taskProcessVariableCount = processVariablesMigrationHelper.getTaskProcessVariableCount(task.getId());
         assertThat(taskProcessVariableCount).isEqualTo(0);
 
-        await()
```

Comment:
does this test cover the bug? if not, can we add a test for it? thanks!

Reply:
I added this for the bug, it was failing with the old approach:
https://github.com/Activiti/activiti-cloud/blob/0c2454ad2e77fa2005c2a88173e873030b35833b/activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/rest/TaskControllerHelperIT.java#L215